NGitLab/IPipelineScheduleClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NGitLab.Models;
+
+namespace NGitLab;
+
+public interface IPipelineScheduleClient
+{
+    /// <summary>
+    /// All the scheduled pipelines of the project
+    /// </summary>
+    IEnumerable<PipelineScheduleBasic> All { get; }
+
+    /// <summary>
+    /// Return the details of single schedule
+    /// </summary>
+    /// <param name="id">Schedule Id</param>
+    /// <returns></returns>
+    PipelineSchedule this[int id] { get; }
+
+    /// <summary>
+    /// Return the details of single schedule
+    /// </summary>
+    /// <param name="id">Schedule Id</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<PipelineSchedule> GetByIdAsync(int id, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// All the scheduled pipelines of the project
+    /// </summary>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    GitLabCollectionResponse<PipelineScheduleBasic> GetAllAsync();
+
+    /// <summary>
+    /// Get all pipelines triggered by a pipeline schedule in a project.
```

Comment:
When documenting a method, use the 3rd person (*descriptive form*, such as `gets`) instead of the 2nd person (*prescriptive form*, such as `get`).

This is a widespread standard, in many languages and in various environments. For example, refer to the following:
- [An example from Microsoft doc](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.getattributes?view=net-6.0#definition)
- [This Google doc](https://developers.google.com/style/reference-verbs)